### DISCUSSION TITLE
Looking for examples of other Green Software / Digital Sustainability podcasts and hosts. Know any?

### DISCUSSION THREAD

**Discussion by mrchrisadams** [Green-Software-Foundation/discussions#157] [Environment Variables]
Hi folks.

I've been hosting Environment Variables podcast for couple of years now, and I know the field of green software / digital sustainability is growing, but finding other podcasts is still harder than it needs to be.

So, I'm starting this thread to:

1. share some I have found that I think are worth your time
2. find other recommendations from others

I'll go first, and link to a few example episodes in each case. I'd be grateful if others follow accordingly

### Green IO - https://greenio.gaelduez.com

This is podcast I listen to the most on this field now. Gael Duez has been growing it with a really good set of guests, and it's not funded by advertising or large corporate sponsor - as far a I know, it's funded directly by listeners, a bit like how the Energy Transition show is funded.

https://greenio.gaelduez.com/e/r87yzqq8-38-building-green-software-with-sara-bergman
https://greenio.gaelduez.com/e/18pk94vn-36-climate-change-challenges-to-data-centers-lessons-from-singapore-with-ps-lee
https://greenio.gaelduez.com/e/1np4552n-24-it-sustainability-in-saas-companies-with-niklas-sundberg-and-tereze-gaile
https://greenio.gaelduez.com/e/0nj2v3zn-23-greenwashing-and-friends-with-harriet-kingaby-and-mathieu-jahnich
https://greenio.gaelduez.com/e/vnw636x8-21-greening-software-101-with-arne-tarara-and-anne-currie

### Zero Downtime - https://www.datacenterdynamics.com/en/podcasts/zero-downtime/

This is more focussed on datacentre infra, but recenty every second post has been abour energy, or climate regulation or had interviews with absolute giants in the field. The recent episode with John Booth was really good

https://www.datacenterdynamics.com/en/podcasts/zero-downtime/episode-54-industry-wide-sustainability-with-john-booth-carbon3it/
https://www.datacenterdynamics.com/en/podcasts/zero-downtime/episode-38-ict-environmentalism-and-the-sustainability-game-with-hunter-vaughan-and-nicole-starosielski/

### The SDIA Podcast

This only has a few, but the episode are goods and give somem good industry context:

https://www.listennotes.com/podcasts/sdia-on-digital-sustainability-sdia-z3UolIZCkm8/

### Heatmap - https://heatmap.news/podcast

It's quite new, but the two hosts on this podcast are long time experts on energy, and know how to find good guests. This one here is probably the best interview I've found going into some of the estimates around energy usage growth from AI, and how to critically engage with them, from Jon Koomey, who has been publishing leading researching this field for more than 20 years:

https://heatmap.news/podcast/shift-key-episode-10-ai-data-centers

### The Data Fix - https://shows.acast.com/the-data-fix

This is produced by Mel Hogan, a critical media studies scholar in Canada. There are a few really good episodes here, and I find it a really helpful balance to the general "we will only talk about good news or how god things _could_ be" tone that many digital sustainability podcasts tend to be. the one with Anne Pasek below in particular is really good for a meta discussion about how we talk about sustainability applied to the digital sector:

https://shows.acast.com/the-data-fix/episodes/thirsty-with
https://shows.acast.com/the-data-fix/episodes/efficient-with-anne-pasek

I know there are more, and as you can see these podcasts don't need to be primarily about digital sustainability, as long as they contain a few really informative, and memorable episodes.

Ok, now your turn!

**Comment by davidkopp** [comment]
Thanks for starting this thread, Chris! As I'm also a big podcast listener, here some more podcasts related to digital sustainability...

### Restart Radio - https://therestartproject.org/podcast/

A podcast about repairing electronic stuff, instead of buying new devices. 

### World Wide Waste with Gerry McGovern - https://www.thisishcd.com/show/world-wide-waste-with-gerry-mcgovern

Gerry McGovern talks with guests about the environmental issues of the world wide web (waste).

### Tech Won’t Save Us - https://techwontsave.us/

Tech Won’t Save Us debates the issues of tech and the tech industry from a political perspective.

### OSS for Climate - https://ossforclimate.sustainoss.org/

A quite new podcast about open source software that have a connection with climate protection.

### SUX - The Sustainable UX Podcast - https://sustainableuxnetwork.com/#podcast

A podcast, as the name implies, about how to make UX more sustainable.

### Green the Web - https://green-the-web.podigee.io/

Another podcast about how to make UX greener.

### Web, But Green! - Wenn Internet auf Nachhaltigkeit trifft - https://dr-beyer.de/news/podcast.php

German-speaking podcast about the environmental issues of the internet.
